HHS/Head-Start-TTADP
# TTAHUB-470: Changes for Program Types in Grant List widget

## Description of change

Change the name of the Program column in the Grants List widget to Program Type(s). Display comma separated list or program type abbreviations.


## How to test

View the Profile page for any Grantee. You should notice that the Program column header has changed and now reflects a list of program abbreviations.


## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-470


## Checklists

### Every PR


~- [ ] API Documentation updated~
~- [ ] Boundary diagram updated~
~- [ ] Logical Data Model updated~
~- [ ] [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions~

### Production Deploy


### After merge/deploy
